src/resources/Catalogs/Catalog.ts

```diff
@@ -36,4 +43,8 @@ export default class Catalog extends Resource {
         return this.api.put<CreateCatalogModel>(`${Catalog.baseUrl}/${catalog.id}`, catalog);
     }
     /* eslint-enable @typescript-eslint/unified-signatures */
+
+    getFieldsSuggestions(query: FieldsSuggestionsQueryModel) {
+        return this.api.post<FieldsSuggestionsModel>(`${Catalog.baseUrl}/fieldsSuggestions`);
```

Comment:
:warning: The query parameter is unused and I _think_ it should be passed as second parameter to post. :sweat_smile:

I think you'll need to change the corresponding unit test to add it as second parameter too.

Reply:
Cool, thanks! I made the change.